std/conv.d

```diff
@@ -5870,8 +5872,12 @@ if ((radix == 2 || radix == 8 || radix == 10 || radix == 16) &&
     assert(toChars(123) == toChars(123));
 
     {
+        assert(toChars!2(cast(ubyte) 0).array == "0");
```

Comment:
BTW you can use e.g. `ubyte(0)` to avoid casts to built in types.

Reply:
Thanks!